### PR TITLE
Kubevirt: do not send to firehose already known template.

### PR DESCRIPTION
**Pull request by yaacov**
Follow-up to #1843, do not send template to firehose.

Ref:
https://github.com/openshift/console/pull/1843#discussion_r299022385

### CODE REVIEW

**Comment by yaacov** [comment]
@mareklibra @vojtechszocs @suomiy @rawagner please review.

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details.tsx [hunk 1]
```diff
@@ -10,43 +10,50 @@ import {
...
 import { VMTemplateResourceSummary, VMTemplateDetailsList } from './vm-template-resource';
 import { DataVolumeModel } from '../../models';
 
-export const VMTemplateDetailsFirehose: React.FC<VMTemplateDetailsFirehoseProps> = ({
-  obj: template,
-}) => {
-  const { name, namespace } = template.metadata;
+const VMTemplateDetailsFirehose: React.FC<VMTemplateDetailsFirehoseProps> = (props) => {
+  const { obj: template, hasDataVolumes } = props;
+  const { namespace } = template.metadata;
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details.tsx]: can we use a selector here?

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details.tsx]: sure, thanks !

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details.tsx]: done

**Pull request merged**